### PR TITLE
Add snippets file for this project

### PR DESCRIPTION
**Pull request by rlh1994**
Add a snippets file for users to simplify writing of documentation and standardise some code blocks, also helps to advertise some features that may not be immeditely obvious or are complex to remember (such as themed images).

### CODE REVIEW

File: .vscode/snowplow-docs.code-snippets [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	// 	],
+	// 	"description": "Log output to console"
+	// }
+	"Header": {
+			"scope": "markdown",
+			"prefix": "head",
+			"body": [
+				"---",
+				"title: \"$1\"",
+				"date: \"$CURRENT_YEAR-$CURRENT_MONTH-$CURRENT_DATE\"",
```
[the remark this answers is not in the document]
Comment (stanch) [reply .vscode/snowplow-docs.code-snippets:24]: I mean the date field

File: .vscode/snowplow-docs.code-snippets [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+			""
+		],
+		"description": "Docusaurus mdx block for tabs and themed images"
+	},
+	"Themed image block": {
+		"scope": "markdown",
+		"prefix": "theme",
+		"body": [
+			"<ThemedImage ",
+				"alt='Docusaurus themed image'",
```
Comment (stanch) [inline .vscode/snowplow-docs.code-snippets:49]: Make the alt a tabstop

File: .vscode/snowplow-docs.code-snippets [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+		"body": [
+			"<TabItem value=\"$1\" label=\"$2\">",
+"",
+			"$3",
+"",
+			"</TabItem>"
+		],
+		"description": "Additional tab item block"
+	},
+}
```
Comment (stanch) [inline .vscode/snowplow-docs.code-snippets:92]: Should we add admonitions (`:::note` etc) as well?

**Review by stanch — CHANGES_REQUESTED** [review]

File: .vscode/snowplow-docs.code-snippets (hunk 1 above)
[the remark this answers is not in the document]
Comment (rlh1994) [reply .vscode/snowplow-docs.code-snippets:24]: Is there anything else we should be putting in the header?

[the remark this answers is not in the document]
Comment (stanch) [reply .vscode/snowplow-docs.code-snippets]: `description` would be a good one for SEO and for the doc cards on index pages

File: .vscode/snowplow-docs.code-snippets [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+	// 	],
+	// 	"description": "Log output to console"
+	// }
+	"Header": {
+			"scope": "markdown",
+			"prefix": "head",
+			"body": [
+				"---",
+				"title: \"$1\"",
+				"sidebar_position: 10",
```
Comment (stanch) [inline .vscode/snowplow-docs.code-snippets:24]: Would change this to `1` — we know know that Docusaurus supports floating point positions, so there is no need to do multiples of 10

Comment (rlh1994) [reply .vscode/snowplow-docs.code-snippets:24]: I understand they support it, I just don't like it 😆 
I'll make it a tab stop, compromise.

**Comment by stanch** [comment]
Just remembered another one to add: https://docusaurus.io/docs/markdown-features#details.
Other than that, looks good! Maybe add a note in the README about the existence of this and how to trigger it in VS Code?

**Review by stanch — APPROVED** [review]

**Pull request merged**